README.md

```diff
@@ -39,6 +60,14 @@ After successfully uploading the pipeline to concourse CI login and unpause it.
 
 See the [integration documentation](README_CICD_INTEGRATION.md) on how to modify the `pipeline.yaml` and config files for your building block.
 
+## Versioning
+
+This template building block uses [bb-gitversion](https://github.com/elbb/bb-gitversion) to compute the semantic version of itself from the git history.
+
+Using `dobi.sh` the version is updated when necessary and published in environment variables. You can use these environment variables in your dobi targets. 
```

Comment:
... and published in different formats e.g. environment variables or plain files containing version version information.

Reply:
how can you use these plain files in dobi? you mean via mount?

Reply:
you'll need them at least for concourse